src/services/DealService.ts

```diff
@@ -64,11 +68,31 @@ export class DealService {
     /**
      * deals will take care of themselves on account changes
      */
-    this.getDeals();
-    this.getDAOsInformation();
+    this.getDealsIPFS();
+    // this.getDAOsInformation();
```

Comment:
@dkent600 @giladt 

Right now, we always load deepDAO information, imo, we should disable?
Reason: I want to prevent unnecessary calls.

This is a side "fix" and could be done separately